lldb/include/lldb/API/SBInstructionList.h

```diff
@@ -54,6 +54,10 @@ class LLDB_API SBInstructionList {
 
   bool GetDescription(lldb::SBStream &description);
 
+  // Writes assembly instructions to `description` with load addresses using
+  // `frame`.
+  bool GetDescription(lldb::SBStream &description, lldb::SBFrame &frame);
```

Comment:
We'd want this API to be as generic as possible. Since you only need the execution context, that means this should take an SBExecutionContext, which you can create from a frame or thread or process.

Reply:
:100: